service/src/main/java/eu/europa/ec/fisheries/uvms/asset/domain/entity/CustomCode.java

```diff
@@ -12,7 +12,7 @@
 @Entity
 @Table(name = "customcode"/*, indexes = { @Index(columnList = "id", name = "customcodes00", unique = true),}*/)
 @NamedQueries({
-        @NamedQuery(name = CUSTOMCODES_GETALLFOR, query = "SELECT m FROM CustomCode m where  upper(m.primaryKey.constant) =: constant"),
```

Comment:
Why is this removed?

Reply:
That is the one I discussed with Jonas and Thomas. That latest chance caused a test to fail. When I used the earlier version, test passed. I was going to ask you about it but apparently, I forgot about it.

Reply:
I fixed the query and working as expected now.